payjoin/src/receive/v2/mod.rs

```diff
@@ -71,12 +73,12 @@ fn subdir_path_from_pubkey(pubkey: &HpkePublicKey) -> ShortId {
 
 /// A payjoin V2 receiver, allowing for polled requests to the
 /// payjoin directory and response processing.
```

Comment:
I think this comment is meant to live with `Receiver`, `NewReceiver` is the persisted variety whose only purpose is to be persisted